sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/encoders/ExpressionEncoder.scala

```diff
@@ -285,7 +285,7 @@ case class ExpressionEncoder[T](
       // transform attributes accessors.
       objDeserializer.transform {
         case UnresolvedExtractValue(GetColumnByOrdinal(0, _),
-            Literal(part: UTF8String, StringType)) =>
```

Comment:
the previous indentation is correct.

Reply:
Oh, I see. (Actually, I never touch it previously. It seems like another Github mistake.)